Updated installation guide after the docker PR (#437)
azerothcore/wiki: docs/Install-with-Docker.md

Now go into the main directory using `cd azerothcore-wotlk`. **All commands will have to be run inside this folder**.

### Installation

Inside your terminal (if you use Windows, use git bash), run the following commands inside the azerothcore-wotlk folder

NOTE: the following procedure uses our acore.sh dashboard, however, these commands are a shortcut of the docker-compose ones.
      you can check the docker-compose commands used in background by running `./acore.sh docker --help` and read the description of each command

**1) Download the client data:**

```
./acore.sh docker client-data
```

NOTE: This command should be executed only at the first installation and when there's a new version of the client-data available

**2) Compile AzerothCore:**
```
./acore.sh docker build
```
This will take a while. Meanwhile you can go and drink a glass of wine :wine_glass:

**3) Run the containers**

```

> ./acore.sh docker start:app

Reviewer: think we should also mention how to run in detached mode.

UPDATE realmlist SET address='<SERVER PUBLIC IP ADDRESS>';
```

### How to run the worlserver with GDB

1. Create a `config.sh` file under the `/conf/` directory of the azerothcore-wotlk repository
2. Add this configuration inside: `AC_RESTARTER_WITHGDB=true`. It will configure the restarter used by our docker services to use GDB instead of the binaries directly

> ### How to use the dev-container

Reviewer 2: what about a short introduction about what the dev-container actually is and how it can be useful when developing?

Feel free to ask questions on [StackOverflow](https://stackoverflow.com/) and link them in the **#support-docker** channel of our [Discord chat](https://stackoverflow.com/questions/tagged/azerothcore). We will be happy to help you!

## FAQ

> *Faq? Faq you!*

Reviewer 2: lol

### Where are the etc and logs folders of my server?

By default they are located in `env/docker/authserver/` and `env/docker/worldserver/`.

### How can I change the docker containers configuration?

> You can copy the file `/conf/dist/.env.docker` within the root folder of the project, renaming the copy in `.env` and editing it accordingly to your needs.

Reviewer 2: not sure about it but to me this feels clearer:
```suggestion
You can copy the file `/conf/dist/.env.docker` to `.env` and place it in the root folder of the project, then edit it according to your needs.
```

**Note** If you've changed your folder name from the default `azerothcore-wotlk` the volume name will be slightly different. To find the new volume name you can use the command `docker volume ls`. The volume should be labelled something along the lines of `xxxx_ac-database`.

> ### MacOSx optimizations

Reviewer 2: it's now `macOS` (lower case `m` and no `X` anymore)

> We optimized the docker-compose file for the OSX by using volumes and the "delegated" strategy.

Reviewer 2: same as above
